slides/discovery-board-stm32mp2/discovery-board-stm32mp2.tex

```diff
@@ -0,0 +1,26 @@
+\begin{frame}
+\frametitle{Supported hardware}
+  STM32MP257f-DK, discovery Kits from STMicroelectronics. 
+  \footnotesize
+  \begin{columns}
+    \column{0.6\textwidth}
+    \begin{itemize}
+    \item STM32MP257 (Dual Cortex-A35 + Cortex-M33) CPU from STMicroelectronics
+    \item 32‑Gbit LPDDR4 DRAM 
+    \item Gigabit Ethernet port
+    \item Two USB 2.0 high speed, USB 3.0 SuperSpeed PD (DRP/DRD) 
+    \item 1 Micro SD slot
+    \item On-board STLINK-V3EC debugger
+    \item Misc: buttons, LEDs, BLE, WIFI
```

Comment:
WiFi doesn't seem to be a "Misc" thing, it warrants its own bullet point. Does it have Bluetooth too?